app/views/home/includes/_how.html.slim

```diff
@@ -0,0 +1,26 @@
+h2 How to create a worker
+ol
+  li
+    h3 Navigate to service directory <span class="code">/service/</span>
+  li
+    h3 Create sandboxed virtual environment (OPTIONAL)
+    pre
+      code.bash
+        | virtualenv -p YOUR_PATH_TO_PYTHON3.6 .
```

Comment:
I made some changes to this to ensure it gets put onto two lines, probably got lost in the rebase: change to:
```
code.bash virtualenv -p YOUR_PATH_TO_PYTHON3.6 .
          . bin/activate # activate the virtualenv
```

Reply:
Is this not correct?

<img width="362" alt="screen shot 2017-11-26 at 7 10 27 pm" src="https://user-images.githubusercontent.com/8942499/33245999-7fce1b52-d2dd-11e7-9049-d31769749253.png">

Reply:
I updated it so this is what it looks like now:

<img width="1086" alt="screen shot 2017-11-26 at 7 32 58 pm" src="https://user-images.githubusercontent.com/8942499/33246284-bdc5beda-d2e0-11e7-9b82-b55ea84b3655.png">

This works?